Source/WebGPU/WGSL/Parser.cpp

```diff
@@ -161,6 +161,20 @@ static bool canContinueRelationalExpression(const Token& token)
     }
 }
 
+static bool canProceedSingularExpression(const Token& token)
```

Comment:
nit: should this be called `canProceedUnaryExpression` to match the rest?

Reply:
Well a unary expression included the operator, ie `-x` is a unary expression, so I named it for the `x`. It should probably be called `canBeginUnaryExpression`.